src/runtime/pay-flow-test.js

```diff
@@ -649,6 +650,35 @@ describes.realWin('PayCompleteFlow', {}, (env) => {
     expect(PayCompleteFlow.waitingForPayClient_).to.be.true;
   });
 
+  it('should have valid flow constructed that triggers the payConfirmOpened callback', async () => {
```

Comment:
Prefer test descriptions that directly state the expected outcome, rather than saying "should...". Example:
'constructs valid flow that triggers the payConfirmedOpened callback'

Or even:
'triggers the payConfirmedOpened callback when dialog opens after flow start'

Also, minor nit: wrap the test description if it exceeds the 80-char line limit.

Reply:
Makes sense! Updated and wrapped description.

> I'm not sure I fully understand the use case for this new callback. Do you need to resize the iframe immediately after it opens?

In Twg code, I wanted to register a handler for this callback:

this.callbacks().setOnPayConfirmOpened((activityIframeView) => {
    activityIframeView.on(SkuSelectedResponse, this.startPayFlow_.bind(this));
    activityIframeView.on(EntitlementsResponse, ...);
    activityIframeView.on(ActionRequest, ...);
    ...
});

For the Twg pay confirm view, we want to attach response listeners from the host because the buy flow can be started again from the pay confirm view.